truss/templates/server/model_wrapper.py

```diff
@@ -139,6 +138,33 @@ def predict(
         finally:
             self._predict_lock.release()
 
+    async def __call__(self, body: Any, headers: Dict[str, str] = None) -> Dict:
+        """Method to call predictor or explainer with the given input.
+
+        Args:
+            body (Any): Request payload body.
+            headers (Dict): Request headers.
+
+        Returns:
+            Dict: Response output from preprocess -> predictor -> postprocess
+        """
+        # TODO(bola): Do we need any of the timing stuff that was dropped here?
```

Comment:
Eventually, it will be valuable for the customer to see from an observability standpoint, but agreed to drop that implementation. When we re-implement, it can be with our own methods.

Reply:
I'm curious why you can't pull all that info from prometheus?

Reply:
Since `enable_latency_logging` was always set to false, I don't think the original timing code was even running. So I don't think this is something we were using or need to come back to. I think it's a legacy way of kserve doing timing that we have a different way of doing.